dev-dotnet/dotnet-sdk/dotnet-sdk-5.0.203.ebuild

```
@@ -0,0 +1,224 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit check-reqs toolchain-funcs flag-o-matic multiprocessing
+
+DESCRIPTION=".NET SDK (includes runtime + aspnet)"
```

Need to check what the canonical style is:
```suggestion
DESCRIPTION=".NET SDK (includes runtime + ASP.NET)"
```